src/main/scala/com/cognite/sdk/scala/common/ReadWritableResource.scala

```diff
@@ -69,29 +54,35 @@ object DeleteByExternalIds {
   implicit val errorOrUnitDecoder: Decoder[Either[CdpApiError, Unit]] =
     EitherDecoder.eitherDecoder[CdpApiError, Unit]
 
-  def deleteByExternalIdsWithIgnoreUnknownIds[F[_]](
+  def deleteByExternalIds[F[_]](
       requestSession: RequestSession[F],
       baseUrl: Uri,
-      externalIds: Seq[String],
-      ignoreUnknownIds: Boolean
+      externalIds: Seq[String]
   ): F[Unit] =
     // TODO: group deletes by max deletion request size
     //       or assert that length of `ids` is less than max deletion request size
-    requestSession.post[Unit, Unit, ItemsWithIgnoreUnknownIds[CogniteId]](
-      ItemsWithIgnoreUnknownIds(externalIds.map(CogniteExternalId.apply), ignoreUnknownIds),
+    requestSession.post[Unit, Unit, Items[CogniteExternalId]](
+      Items(externalIds.map(CogniteExternalId.apply)),
       uri"$baseUrl/delete",
       _ => ()
     )
+}
 
-  def deleteByExternalIds[F[_]](
+trait DeleteByCogniteIds[F[_]]
+    extends DeleteByIdsWithIgnoreUnknownIds[F, Long]
+    with DeleteByExternalIdsWithIgnoreUnknownIds[F] {
+  def deleteWithIgnoreUnknownIds(ids: Seq[CogniteId], ignoreUnknownIds: Boolean = false): F[Unit]
```

Comment:
```suggestion
  def deleteWithIgnoreUnknownIds(ids: Seq[CogniteId], ignoreUnknownIds: Boolean = false): F[Unit]
  
  def deleteByIds(ids: Seq[Long], ignoreUnknownIds: Boolean = false) =
      deleteWithIgnoreUnknownIds(ids.map(CogniteInternalId(_)), ignoreUnknownIds)
```


What I meant is to add the default implementation here, so have less of the duplicated code. But if you don't like it, never mind, it's no big deal

Reply:
it's not I don't like it but I didn't understand it 😆 , I got it know, and did the same thing for `deleteByExternalIds`, much less boilerplate code now, thanks ;)